source/algorithms.tex

```diff
@@ -88,6 +88,14 @@
   \tcode{Forward\-Iterator2},
   the template argument shall meet the
   \oldconcept{ForwardIterator} requirements\iref{forward.iterators}.
+\item
+  If an algorithm’s template parameter is named
+  \tcode{NoThrowForwardIterator},
+  the template argument shall meet the
+  \oldconcept{ForwardIterator} requirements\iref{forward.iterators}, and
+  is required to have the property that no exceptions are thrown
+  from increment, assignment, comparison, or
+  indirection through valid iterators.
```

Comment:
The wording here doesn't make sense, can it be fixed?

Reply:
My guess is you're reading this as "comparison through valid iterators".

Reply:
Fixed in separate commit so @zygoloid can omit it if it's too much editorial fixup.